.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
+        os: ['windows-latest', 'ubuntu-latest', 'macos-13']
```

Comment:
Update GitHub required actions from `macos-latest` to `macos-13` when ready to merge